src/screens/AllVoucherScreen/components/MoreActionModalize.tsx

```
@@ -0,0 +1,220 @@
+  TouchableOpacity,
+  View,
+} from 'react-native';
+import Feather from 'react-native-vector-icons/Feather';
+import MaterialCommunityIcons from 'react-native-vector-icons/MaterialCommunityIcons';
+import BottomSheet from '@/components/BottomSheet';
+import Toast from '@/components/Toast';
+import { APP_EVENTS } from '@/utils/constants';
+import useCustomTheme, { ThemeProps } from '@/utils/theme';
+import { useNavigation } from '@react-navigation/native';
+
+const _MoreActionBottomSheet: React.ForwardRefRenderFunction<{}, { open: (data: any) => void;}> = ({downloadFile, shareFile }, ref) => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)